ext/mbstring/php_mbregex.c

```diff
@@ -1109,6 +1109,7 @@ static void _php_mb_regex_ereg_replace_exec(INTERNAL_FUNCTION_PARAMETERS, OnigOp
 				} else {
 					if (!EG(exception)) {
```

Comment:
I believe this code is even unreachable. zend_call_function only returns FAILURE when !EG(active), which should not be possible to hit from here. And retval is only UNDEF if an exception _did_ happen.

Reply:
After double checking I think I'm right. You can drop this else branch in master.